UnitTests/Code/TestCard.cs

```diff
@@ -1,8 +1,7 @@
 ﻿
-using System.Globalization;
 using System;
 using System.Diagnostics;
-using Media.Codec;
+using System.Globalization;
```

Comment:
Is it needed?

Reply:
I've just run Visual Studio automatic code cleanup.

`CultureInfo.InvariantCulture` is in `System.Globalization` namespace.